web/templates/settings/playground_settings_admin.hbs

```diff
@@ -71,9 +71,9 @@
         <div class="progressive-table-wrapper" data-simplebar>
             <table class="table table-condensed table-striped wrapped-table admin_playgrounds_table">
                 <thead class="table-sticky-headers">
-                    <th class="active" data-sort="pygments_language">{{t "Language" }}</th>
-                    <th data-sort="playground_name">{{t "Name" }}</th>
-                    <th data-sort="url">{{t "URL prefix" }}</th>
```

Comment:
Is `data-sort="url"` a mistake from before? I don't see any sort function named `url` prior to this change.

Reply:
Yes, that indeed was a mistake from before we didn't have a sorting function for "url" before.

Also, I think this is another reason to pass every sorting functions explicitly whether be it generic sorts or custom sorts to avoid having silent bugs like this. (Context - #25326)